src/node.cc

```diff
@@ -584,6 +584,63 @@ const char *signo_string(int signo) {
   }
 }
 
+// These are all flags available for use with NODE_OPTIONS.
+//
+// Disallowed flags:
+//  These flags cause Node to do things other than run scripts:
+//    --version / -v
+//    --eval / -e
+//    --print / -p
+//    --check / -c
+//    --interactive / -i
+//    --prof-process
+//    --v8-options
+//  These flags are disallowed because security:
+//    --preserve-symlinks
+static const char* environment_flags[] = {
```

Comment:
Since you're here, can you make this `static const char* const environment_flags[] = ...`?

Reply:
done